.travis.yml

```diff
@@ -63,11 +61,11 @@ matrix:
       addons:
         hosts:
           - mysql2gem.example.com
-        apt:
-          packages:
-          - mysql-server-5.7
-          - mysql-client-core-5.7
-          - mysql-client-5.7
+        # apt:
+        #   packages:
+        #   - mysql-server-5.7
+        #   - mysql-client-core-5.7
+        #   - mysql-client-5.7
```

Comment:
Probably. I've mentioned the upstream issue in the commit message as a bread crumb for future work.